demo/app/components/cohort-date-range/cohort-date-range.module.ts

```diff
@@ -0,0 +1,31 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import {
+  FormsModule,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import { TsCardModule } from '@terminus/ui/card';
+import { TsCohortDateRangeModule } from '@terminus/ui/cohort-date-range';
+import { TsSpacingModule } from '@terminus/ui/spacing';
+
+import {CohortDateRangeComponent} from "./cohort-date-range.component";
+import {TsDateRangeModule} from "@terminus/ui/date-range";
+import {CohortDateRangeRoutingModule} from "./cohort-date-range-routing.module";
```

Comment:
Curious as to why linting is passing in CI but we don't have the bracket spacing consistent 😕